app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+end
+
+# A method that returns the 3 most recent posts for a given user.
+def recent_3_posts
+  post.last(3)
+end
```

Comment:
- [ ] Please setup associations between models like for user model you should add association between user, `posts`, `comments`, `likes`

Reply:
Done.